### PR TITLE
adding chart for prometheus-couchdb-exporters

### PR DESCRIPTION
**Pull request by gkarthiks**
**What this PR does / why we need it**:
Adding new chart for the CouchDB metrics exporter based on [couchdb-prometheus-exporter)](https://github.com/gesellix/couchdb-prometheus-exporter)

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
N/A

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by gkarthiks** [comment]
Could you please take a look at this PR.
@arschles @fibonacci1729

File: stable/prometheus-couchdb-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1beta2
```
Comment (davidkarlsen) [inline stable/prometheus-couchdb-exporter/templates/deployment.yaml]: use v1

Comment (gkarthiks) [reply stable/prometheus-couchdb-exporter/templates/deployment.yaml]: @davidkarlsen Updated to `apps/v1`.

**Comment by gkarthiks** [comment]
@davidkarlsen  added the rbac.

**Pull request merged**